sklearn/metrics/pairwise.py

```diff
@@ -1187,29 +1187,32 @@ def linear_kernel(X, Y=None, dense_output=True):
 
 
 def polynomial_kernel(X, Y=None, degree=3, gamma=None, coef0=1):
-    """
-    Compute the polynomial kernel between X and Y::
-
+    """Compute the polynomial kernel between X and Y.
         K(X, Y) = (gamma <X, Y> + coef0)^degree
 
     Read more in the :ref:`User Guide <polynomial_kernel>`.
 
     Parameters
     ----------
     X : ndarray of shape (n_samples_X, n_features)
+        A feature array.
```

Comment:
We can be a bit more informative here
```suggestion
        An array where each row is a sample and each column is a feature.
```

Reply:
I requested this formulation in other PRs to keep local consistency with other kernel and distances functions in the file because this is what has been done in previous PRs for numpydoc validation